src/chain/keysinterface.rs

```diff
@@ -122,12 +124,13 @@ impl ChannelKeys {
 	}
 }
 
-/// Simple CKeysInterface implementor that takes a 32-byte seed for use as a BIP 32 extended key
+/// Simple KeysInterface implementor that takes a 32-byte seed for use as a BIP 32 extended key
 /// and derives keys from that.
 ///
 /// Your node_id is seed/0'
 /// ChannelMonitor closes may use seed/1'
 /// Cooperative closes may use seed/2'
+/// Per-channel root secret may use 32 first bytes of pubkey from seed/3'
```

Comment:
This bit I deliberately left out cause per-channel root secret is a somewhat internal-facing thing, users never need to care about such keys.